cms/djangoapps/contentstore/tests/test_import.py

```diff
@@ -84,13 +84,14 @@ def test_unicode_chars_in_course_name_import(self):
         # Test that importing course with unicode 'id' and 'display name' doesn't give UnicodeEncodeError
         """
         module_store = modulestore()
-        course_id = SlashSeparatedCourseKey(u'Юникода', u'unicode_course', u'échantillon')
+        course_id = module_store.make_course_key(u'Юникода', u'unicode_course', u'échantillon')
         import_from_xml(
             module_store,
             self.user.id,
             TEST_DATA_DIR,
             ['2014_Uni'],
-            target_course_id=course_id
+            target_course_id=course_id,
+            create_course_if_not_present=True
```

Comment:
Note: with this change, this test fails if run with old mongo as the default (see https://build.testeng.edx.org/job/edx-platform-all-tests-pr/271/SHARD=1,TEST_SUITE=unit,label_exp=jenkins-worker/testReport/junit/cms.djangoapps.contentstore.tests.test_import/ContentStoreImportTest/test_unicode_chars_in_course_name_import/).

This is the only test failure with old mongo as the default.